launch/.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 100
```

Comment:
edit: later somewhere it's changed to 99. So my question is just generally if you could clarify why you chose to add a custom one instead of using the default ament_flake8 one?

Reply:
I configured my editor to use the ament_flake8 config file as the default. I figured this was the expected workflow, so we can configure it in one place, as opposed to adding one to each repo (unless it needs to be customised for some reason). @dirk-thomas might have an opinion but there's no need to block the PR on it

Reply:
That's fine, I can remove it. I just found it useful, but I agree that it has the potential to get out of sync with `ament_flake8`'s config.